### PR TITLE
add recipe pywps 4.x

### PR DESCRIPTION
**Pull request by cehbrecht**
this recipe needs flufl.enum (PR #2293) for python 2.7 builds.

### CODE REVIEW

**Comment by cehbrecht** [comment]
does someone know why the pywps build fails on windows (AppVeyor) for py27? 
Do i need to change something  in the recipe?

**Comment by cehbrecht** [comment]
the windows failure for py27 looks like this issue:

https://github.com/conda/conda-build/issues/820

But it is supposed to be fixed.

**Comment by cehbrecht** [comment]
skipped the windows build ... pywps is only tested on linux and osx.

File: recipes/pywps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "pywps" %}
+{% set version = "4.0.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/geopython/pywps
+    git_tag: {{ version }}
```
Comment (ocefpaf) [inline recipes/pywps/meta.yaml]: Please change this to a tarball "format" like:

```yaml
source:
  fn: compliance-checker-{{ version }}.tar.gz
  url: https://github.com/ioos/compliance-checker/archive/{{ version }}.tar.gz
  sha256: 2556b724ec639b4ce9b607c764682c7cfd2aa2d904867d7ea9d2c111a52787d3
```

File: recipes/pywps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+build:
+    number: 0 
+    skip: True  # [win]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - owslib
```
Comment (ocefpaf) [inline recipes/pywps/meta.yaml]: When using setuptools and `--single-version-externally-managed` you probably don't need the `run` dependencies at build time. Try removing,

```
        - owslib
        - jsonschema
        - werkzeug
        - lxml
        - sqlalchemy
        - flufl.enum  # [py27]
        - python-dateutil
```

PS: if the package imports itself in the `setup.py` that may not be true and you will need to leave those there.

**Pull request merged**